conda-build/common/delete_old_night_packages.py

```diff
@@ -0,0 +1,23 @@
+import subprocess
```

Comment:
Hmm, looks the pre-commit hooks aren't enabled on this directory!  Can you run black on this manually for now?

Reply:
@erikwijmans Fixed now in another PR. Was a typo in the config.